common/test/java/com/couchbase/lite/CollectionListenerTest.kt

```diff
@@ -53,6 +55,114 @@ class CollectionListenerTest : BaseCollectionTest() {
         testCollectionChangeListener(testSerialExecutor)
     }
 
+    // Test that addChangeListener to a deleted collection get a warning message and doesn't throw exception
+    @Ignore("Native crash")
+    @Test
+    fun testAddChangeListenerToDeletedCollection() {
+        baseTestDb.deleteCollection(testColName, testScopeName)
+        testCollection.addChangeListener(testSerialExecutor) {}
+    }
+
+
+    // Test that addChangeListener to a collection deleted from a different db instance get a warning message and doesn't throw exception
+    @Ignore("Native crash")
```

Comment:
The problem with this is that it isn't actionable: there's no real way to tell when the @Ignore can be removed.  I think the comment on the Ignore should be a ticket number....